### DISCUSSION TITLE
How did you manage that the initial page load is so fast

### DISCUSSION THREAD

**Discussion by razfazz** [champ96k/champ96k.com#7] [General]
Normally, Flutter Web projects are very slow when first loaded. Your page is very fast and I saw that your main.dart.js is also only 1.4 megabytes. how did you manage that?

**Comment by champ96k** [comment]
@razfazz  Currently I’m rendering using `CanvasKit renderer`, As you can see everything is static so that's the reason the initial loading time is almost zero. 

For more info you can read about rendering flutter web app [Documentation link](https://docs.flutter.dev/platform-integration/web/renderers), [Medium article](https://medium.com/bina-nusantara-it-division/flutter-web-renderers-which-one-should-i-use-html-or-canvaskit-renderer-b295e6bc995b), May be you can play around it.
